test.sh

```diff
@@ -29,17 +27,27 @@ else
     popd
 fi  
 
-# build latest test images
+# Build latest test images
 docker-compose build
 
-# unpack latest build of buildpack
+# Unpack latest build of buildpack
 docker-compose run --rm tester bash ./unpack.sh
 
-# run tests against latest build of buildpack
-docker-compose run --rm \
- -w "$BUILDPACK_ROOT_DIR/ci" \
- tester cucumber \
- --format pretty \
- --format junit \
- --out ./features/reports \
- ${INTEGRATION_TAG}
+if [ -n "$CF_API_ENDPOINT" ]; then
+     # Run tests against latest build of buildpack
+     docker-compose run --rm \
+         -w "$BUILDPACK_ROOT_DIR/ci" \
+         tester cucumber \
+         --format pretty \
+         --format junit \
+         --out ./features/reports
+else
+    # Skip the integration tests if the Summon variables are not present
+    docker-compose run --rm \
+        -w "$BUILDPACK_ROOT_DIR/ci" \
+        tester cucumber \
+        --format pretty \
+        --format junit \
+        --out ./features/reports \
+        --tags 'not @integration'
```

Comment:
why did you change the syntax from `~@integration`?

Reply:
That is the syntax required for newer versions of cucumber. Additionally, if you use multi-word tag statements, such as in this case, cucumber requires it to be inside single quotes. There's no easy way to store single quotes inside a string and then pass them into this docker-compose statement, and this is the only solution I could find after spending a good deal of time on it.